CMakeLists.txt

```diff
@@ -434,7 +434,7 @@ if(WITH_CORE)
     find_package(PDAL) # PDAL provider
```

Comment:
Should this be `REQUIRED`?
If `WITH_PDAL` is set to true but it's not found, app will fail to build. Better fail early in this case so I can install missing libraries (or turn off pdal).

Reply:
For reference, the old behavior was that if `WITH_PDAL` was set but pdal was not found it would just build without pdal. I very much prefer the new behavior.

Reply:
fixed